torch/csrc/jit/passes/onnx/initializer_peephole.cpp

```diff
@@ -0,0 +1,136 @@
+#include <torch/csrc/jit/passes/onnx/initializer_peephole.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+#include <torch/torch.h>
+
+#include <c10/util/Optional.h>
+#include <algorithm>
+
+namespace torch {
+namespace jit {
+
+namespace onnx {
+using namespace ::c10::onnx;
+}
+
+std::vector<at::Tensor> getValues(
```

Comment:
If this is the same function as in constant_folding_pass, please move it to a helper file to avoid duplicates.

Reply:
it is not the same, but it is very simililar